packages/admin-next/dashboard/src/v2-routes/promotions/promotion-list/promotions-list.tsx

```diff
@@ -0,0 +1,22 @@
+// TODO: This needs to be replaced with promotions
+// Question: Where is this being initialized?
+import after from "medusa-admin:widgets/discount/list/after"
+import before from "medusa-admin:widgets/discount/list/before"
```

Comment:
q: how can i create injections for promotions?

Reply:
think that it would be fine to postpone this for a follow up ticket since zones for v1 also need to be revisited